tests/hash-graph-http/tests/ambiguous.http

```diff
@@ -60,8 +60,7 @@ X-Authenticated-User-Actor-Id: {{account_id}}
         "$id": "http://localhost:3000/@alice/types/data-type/meter/v/1",
         "title": "Meter",
         "type": "number",
-        "description": "A unit of length",
-        "minimum": 0
```

Comment:
Out of interest, why did these need to be removed?

Reply:
They should be inherited from the parent anyway. I don't use the constraint in this file, however, I think it's better practice to not repeat every constraint on child data types if not needed.